places-solution-integrations/analytics/places-trigger.md

```diff
@@ -1,68 +1,80 @@
+3. [Add any other **Conditions** required to trigger the action (optional)](#add-conditions)
+4. [Select the **Action** to send data to Analytics and define your payload](#define-the-action)
 
 ## Create a Rule
 
-In the `Rules` tab, click the `Create New Rule` button.
+1. On the **Rules** tab, click **Create New Rule**.
 
-If you don't have existing rules for this property, the button will be in the middle of the screen:
-
-<img src="../../.gitbook/assets/integrations/addFirstRule.png" width="800" /><br />
-
-If your property already has rules, the button will be in the top right of the screen:
-
-<img src="../../.gitbook/assets/integrations/addRuleFromList.png" width="800" /><br />
+Remember the following information:
+- If you do not have existing rules for this property, the button will be in the middle of the screen.
+- If your property has rules, the button will be in the top right of the screen.
 
 ## Select an Event
 
-First, you can name your rule. We've named this rule "Send Data to Analytics".
+1. Give your rule a meaningful name so it will be easily recognizable in your list of Rules. In this example, the Rule is named **Send Data to Analytics**.
 
-Next, click the "Add" button under the "Events" section:
+2. In the **Events** section, click **Add**.
 
-<img src="../../.gitbook/assets/integrations/analytics/pt-addEvent.png" width="800" /><br />
+3. From the **Extension** drop-down list, select **Places**.
 
-Select `Places` (or `Places - Beta`) from the "Extension" dropdown, then select `Enter POI` from the "Event Type" dropdown. Hit the "Keep Changes" button.
+4. From the **Event Type** drop-down list, select **Enter POI**.
+
+5. Click **Keep Changes**.
 
 <img src="../../.gitbook/assets/integrations/analytics/pt-selectEvent.png" width="800" /><br />
 
 ## Add Conditions
 
-If you wish to add `Conditions` to your rule, complete this step. If you don't, skip to [Define the Action](#define-the-action).
+{% hint style="important" %}
+Complete this step if you want to add Conditions to your rule. Otherwise, skip to [Define the Action](#define-the-action).
+{% endhint %}
+
+In this example, a Condition is created that causes the Rule to trigger only when the Current POI's name equals **My POI**.
 
-Click the "Add" button under the "Conditions" section:
+1. Under the **Conditions** section, click **Add**.
 
-<img src="../../.gitbook/assets/integrations/analytics/pt-addConditions.png" width="800" /><br />
+2. From the **Extension** drop-down list, select **Places**.
 
-Select `Places` (or `Places - Beta`) from the "Extension" dropdown, and choose from the "Condition Type" dropdown which value you wish to make a condition on. When you're done, hit the "Keep Changes" button.
+3. From the **Condition Type** drop-down list, select **Name**.
 
-In our example, we are making a condition that the Current POI's name must equal "My POI":
+4. In the window on the right, enter **My POI** in the text field.
```

Comment:
Please update to:
"4. In the window on the right, in the text field, enter **My POI** ."